### DISCUSSION TITLE
AvatarGroup should also apply `colorScheme` and `variant` to the children

### DISCUSSION THREAD

**Discussion by isBatak** [chakra-ui/chakra-ui#8385] [Ideas] [answered]
### Description

AvatarGroup is only applying `size` props to children Avatar.
https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/src/avatar/avatar-group.tsx#L82

Add support for passing `colorScheme` and `variant`

### Problem Statement/Justification

`colorScheme` and `variant` are not passed down to the Avatar component.
```ts
<Avatar.Group size='md' colorScheme="blue" varaint="square">
  <Avatar name='Ryan Florence' src='https://bit.ly/ryan-florence' />
  <Avatar name='Segun Adebayo' src='https://bit.ly/sage-adebayo' />
  <Avatar name='Kent Dodds' src='https://bit.ly/kent-c-dodds' />
  <Avatar name='Prosper Otemuyiwa' src='https://bit.ly/prosper-baba' />
  <Avatar name='Christian Nwamba' src='https://bit.ly/code-beast' />
</Avatar.Group>
```

### Proposed Solution or API

This should be supported
```ts
<Avatar.Group size='md' colorScheme="blue" varaint="square">
  <Avatar name='Ryan Florence' src='https://bit.ly/ryan-florence' />
  <Avatar name='Segun Adebayo' src='https://bit.ly/sage-adebayo' />
  <Avatar name='Kent Dodds' src='https://bit.ly/kent-c-dodds' />
  <Avatar name='Prosper Otemuyiwa' src='https://bit.ly/prosper-baba' />
  <Avatar name='Christian Nwamba' src='https://bit.ly/code-beast' />
</Avatar.Group>
```

**Comment by imMatheus** [comment]
I can understand wanting `variant` as passed prop, but what would `colorScheme` get you? Im just trying to understand what use case `colorScheme` has for the `Avatar` component? 🤔

**Reply by isBatak** [reply]
You can override Avatar theme/recipe and use colorScheme/colorPalette to style some additional avatar features like border color or different color combinations for background and initials color.
For example:
![image](https://github.com/chakra-ui/chakra-ui/assets/4128883/9795bcc9-782d-4497-b1c8-a3896a27345d)

**Comment by segunadebayo** [comment, marked as answer]
`colorScheme` is gone. Pass `colorPalette` on `AvatarGroup`.

It flows to the children through the avatar recipe, so you do not set it on each `Avatar`.
